packages/legacy/core/App/screens/RenameContact.tsx

```diff
@@ -0,0 +1,161 @@
+import { useConnectionById } from '@aries-framework/react-hooks'
+import { useNavigation } from '@react-navigation/core'
+import { StackScreenProps } from '@react-navigation/stack'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+
+import ButtonLoading from '../components/animated/ButtonLoading'
+import Button, { ButtonType } from '../components/buttons/Button'
+import LimitedTextInput from '../components/inputs/LimitedTextInput'
+import { InfoBoxType } from '../components/misc/InfoBox'
+import PopupModal from '../components/modals/PopupModal'
+import KeyboardView from '../components/views/KeyboardView'
+import { DispatchAction } from '../contexts/reducers/store'
+import { useStore } from '../contexts/store'
+import { useTheme } from '../contexts/theme'
+import { ContactStackParams, Screens } from '../types/navigators'
+import { testIdWithKey } from '../utils/testable'
+
+type ErrorState = {
+  visible: boolean
+  title: string
+  description: string
+}
+
+type RenameContactProps = StackScreenProps<ContactStackParams, Screens.RenameContact>
+
+const RenameContact: React.FC<RenameContactProps> = ({ route }) => {
+  if (!route?.params) {
```

Comment:
I wonder if this is needed? It's a pattern we do elsewhere but I think TypeScript won't let this be passed is as empty AFAIK.

Reply:
Will remove